pkg/securelaunch/eventlog/eventlog.go

```diff
@@ -51,19 +69,16 @@ func parseEvtLog(evtLogFile string) ([]byte, error) {
 	return []byte(w.String()), nil
 }
 
-/*
- * Persist uses tpmtool to parse event logs generated by the kernel into
- * human readable format, mounts the target disk and store the result in the file on
- * target disk.
- *
- * The location of the file on disk is specified in policy file by Location tag.
- * returns
- * - error if mounting the disk fails __OR_ writing to location on disk fails.
- */
-func (e *EventLog) Persist() error {
+// Parse uses tpmtool to parse event logs generated by the kernel into
+// human readable format, and queues to data to persist queue .
+//
+// The location of the file on disk is specified in policy file by Location tag.
+// returns
+// - error if parsing eventlog fails or user enters incorrect format for input.
+func (e *EventLog) Parse() error {
 
 	if e.Type != "file" {
-		return fmt.Errorf("unsupported eventlog type exiting")
+		return fmt.Errorf("Unsupported eventlog type exiting")
```

Comment:
https://github.com/golang/go/wiki/CodeReviewComments#error-strings

Reply:
This is interesting.. last time i submitted the PR the golang-ci clean-code script caught this. This time it didn't ...perhaps something changed.
Anyway thanks for pointing this out...I have an internal lint"ing" script i run and i just added "gometalinter --vendor pkg/securelaunch/... --enable=golint -I "error strings should not be capitalized"" to it